pkg/app/app_reconcile_test.go

```diff
@@ -250,7 +250,7 @@ func (f FakeComponentInfo) KappControllerVersion() (semver.Version, error) {
 	return f.KCVersion, nil
 }
 
-func (f FakeComponentInfo) KubernetesVersion(serviceAccountName string, specCluster *v1alpha1.AppCluster, objMeta *metav1.ObjectMeta) (semver.Version, error) {
+func (f FakeComponentInfo) KubernetesVersion(_ string, _ *v1alpha1.AppCluster, _ *metav1.ObjectMeta) (semver.Version, error) {
```

Comment:
[FYI] Golang lint v1.52.2 throwing error when function parameters are unused - 
` unused-parameter: parameter 'serviceAccountName' seems to be unused, consider removing or renaming it as _ (revive)`